### ISSUE TITLE
No HTML Weppage

### ISSUE THREAD

**Issue by PurHur** [pmaasz/The-Anti-Social-Network#1]
I dont want to surf on a html document. Can this project please only render PDFs as a document?

**Comment by mrcodefinger** [comment]
Why PDF and not Open Document Format?

**Comment by PurHur** [comment]
@mrcodefinger PDF is the best supported document format for all standard webbrowsers. I guess the IE6 has problems directly showing a odf file.

**Comment by mrcodefinger** [comment]
Well, i just thought because PDF is no open source.

**Comment by PurHur** [comment]
OpenSource is bad

**Comment by jsmmo** [comment]
Also this could be used to close the source of the html (frontendcode) of this project. AMAZING

**Comment by pmaasz** [comment]
Yeah closed source is the best. We could publish it here but close it off so that we can sue everybody that copies the code. I smell big bucks coming boys.

**Comment by pmaasz** [comment]
This issue has no impact anymore

**Comment by jsmmo** [comment]
Why

pmaasz <notifications@github.com> schrieb am Sa., 14. Dez. 2019, 12:51:
